ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -219,6 +219,29 @@ router bgp {{ router_bgp.as }}
 {%             endif %}
    {{ maximum_routes_cli }}
 {%         endif %}
+{%         if peer_group.missing_policy is arista.avd.defined %}
+{%             for direction in ['in', 'out'] %}
+{%                 set dir = 'direction_' ~ direction %}
+{%                 set policy = peer_group.missing_policy[dir] %}
+{%                 if policy.action is arista.avd.defined %}
+{%                     set missing_policy_cli = "neighbor " ~ peer_group.name ~ " missing-policy address-family all" %}
+{%                     if policy.include_community_list is arista.avd.defined(true) or policy.include_prefix_list is arista.avd.defined(true) or policy.include_sub_route_map is arista.avd.defined(true) %}
+{%                         set missing_policy_cli = missing_policy_cli ~ " include" %}
+{%                         if policy.include_community_list is arista.avd.defined(true) %}
+{%                             set missing_policy_cli = missing_policy_cli ~ " community-list" %}
+{%                         endif %}
+{%                         if policy.include_prefix_list is arista.avd.defined(true) %}
+{%                             set missing_policy_cli = missing_policy_cli ~ " prefix-list" %}
+{%                         endif %}
+{%                         if policy.include_sub_route_map is arista.avd.defined(true) %}
```

Comment:
We don't have to check arista.avd.defined again since we are checking in line no. 228
```suggestion
{%                         if policy.include_community_list == True %}
{%                             set missing_policy_cli = missing_policy_cli ~ " community-list" %}
{%                         endif %}
{%                         if policy.include_prefix_list == True %}
{%                             set missing_policy_cli = missing_policy_cli ~ " prefix-list" %}
{%                         endif %}
{%                         if policy.include_sub_route_map == True %}
```